realm/realm-library/src/main/java/io/realm/Realm.java

```diff
@@ -1586,6 +1588,18 @@ public void run() {
         return new RealmAsyncTaskImpl(pendingTransaction, asyncTaskExecutor);
     }
 
+    /**
+     * Invokes transaction on worker thread and returns Completable
```

Comment:
`Invokes the transaction on a worker thread wrapping the result as a {@link Completable}.`